web/src/user_status_ui.js

```diff
@@ -161,7 +161,7 @@ function user_status_post_render() {
 
     $("#clear_status_message_button").on("click", () => {
         clear_message();
-        set_selected_emoji_info();
+        set_selected_emoji_info({});
```

Comment:
This is wrong. We assign a specific meaning to `emoji_info === undefined` in several places, which is different from `{}`. The type that describes this is `UserStatusEmojiInfo | undefined`, not `Partial<UserStatusEmojiInfo>`.

Reply:
Never mind, maybe this particular change doesn’t affect anything. (But `UserStatusEmojiInfo | undefined` or `UserStatusEmojiInfo | null` is still cleaner and we should migrate to that.)